fdb-record-layer-spatial/fdb-record-layer-spatial.gradle

```diff
@@ -47,39 +47,29 @@ dependencies {
     testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:${junitVersion}"
 }
 
-def skipFDB = System.getenv('SKIP_FDB_TESTS') != null && System.getenv('SKIP_FDB_TESTS') == 'true'
-def skipSlow = System.getenv('SKIP_SLOW_TESTS') != null && System.getenv('SKIP_SLOW_TESTS') == 'true'
-test {
-    useJUnitPlatform {
-        if (skipFDB) {
-            excludeTags 'RequiresFDB'
-        }
-        if (skipSlow) {
-            excludeTags 'Slow'
-        }
-    }
-}
-
 def geonames_files = ['countryInfo.txt', 'cities15000.zip', 'shapes_all_low.zip']
 
 task downloadGeonames(type: de.undercouch.gradle.tasks.download.Download) {
     // src geonames_files.collect { file -> "http://download.geonames.org/export/dump/${file}" }
     /* some times geonames times out or is otherwise unreliable, move files to
     an s3 bucket that we control for more consistent download behavior */
     src geonames_files.collect { file -> "https://fdb-record-layer.s3.amazonaws.com/geonames/${file}" }
-    dest buildDir
+    dest layout.buildDirectory.dir("geonames").get().asFile
     overwrite false         // Don't need the very latest.
 }
 
 task unzipGeonames(dependsOn: downloadGeonames, type: Copy) {
-    into buildDir
+    into layout.buildDirectory.dir("geonames")
```

Comment:
This was changed so that gradle no longer considers the entire `.out` directory as the output of `unzipGeonames`, which was causing a bunch of warnings.